api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java

```diff
@@ -16,6 +16,7 @@
 import java.util.Optional;
```

Comment:
Tested by running describe-runtimes.

delete-runtimes is broken here, but that is also true on main :(